lib/moonshot.rb

```diff
@@ -43,6 +43,8 @@ module Plugins # rubocop:disable Documentation
 ].each { |f| require_relative "moonshot/#{f}" }
 
 # Bundled plugins
+# rubocop:disable Style/WordArray
```

Comment:
I disabled this check here to not break the style of using an array here. I think this would look odd. :P
```
%w(
  'backup'
  'cleaner'
).each { |p| require_relative "plugins/#{p}" }
```